exe/ruby-lsp

```diff
@@ -139,9 +139,9 @@ if options[:doctor]
 
   puts "Globbing for indexable files"
 
-  index.configuration.indexables.each do |indexable|
-    puts "indexing: #{indexable.full_path}"
-    index.index_single(indexable)
+  index.configuration.indexables.each do |uri|
```

Comment:
Not a blocker for this, but I think `indexable_uris` would be clearer. (`indexable` sounds like a module/interface).

Reply:
https://github.com/Shopify/ruby-lsp/pull/2924